docs/quick-local-deployment.md

```
@@ -191,12 +195,15 @@ Quay and Clair can also be run as services on a Kubernetes cluster.  This is bec
 ## Troubleshooting
 
 ### I need to change my Quay configuration!
-This can be done by uploading your config tarball back into the ConfigTool:
+Edit `config.yaml` directly in `$QUAY/config`, then restart the Quay container. To enable Clair security scanning on a standalone deployment, see [clair-standalone-configuration.md](clair-standalone-configuration.md).
+
+Optionally validate changes with the config-tool CLI:
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)